### PR TITLE
Replace deprecated firewallctl command

### PR DESCRIPTION
**Pull request by teclator**
- https://bugzilla.suse.com/show_bug.cgi?id=1093111

firewalldstate

### CODE REVIEW

File: library/network/src/lib/y2firewall/firewalld/api.rb [hunk 1]
```diff
@@ -105,7 +105,7 @@ def disable!
 
       # @return [Boolean] The firewalld service state (exit code)
```
Comment (mvidner) [inline library/network/src/lib/y2firewall/firewalld/api.rb]: Let's fix the API docs to match the implementation

**Comment by teclator** [comment]
It was reported for TW but checking that in SLES we already ship firewalld 5.2 then it probably make sense to change the branch to GA

File: library/network/src/lib/y2firewall/firewalld/api.rb [hunk 2]
```diff
@@ -105,7 +105,7 @@ def disable!
...
       def state
-        case Yast::Execute.on_target("firewallctl", "state", allowed_exitstatus: [0, 252])
+        case Yast::Execute.on_target("firewall-cmd", "--state", allowed_exitstatus: [0, 252])
```
Comment (mvidner) [inline library/network/src/lib/y2firewall/firewalld/api.rb]: (I very much like the small change but I have no idea if it is correct. I really should get my own 15.0 for testing.)

Comment (teclator) [reply library/network/src/lib/y2firewall/firewalld/api.rb]: I have it and tested manually.

Comment (teclator) [reply library/network/src/lib/y2firewall/firewalld/api.rb]: Will adapt the documentation:

http://www.firewalld.org/documentation/man-pages/firewall-cmd.html

Comment (teclator) [reply library/network/src/lib/y2firewall/firewalld/api.rb]: firewalldstate

**Pull request merged**